test/Base_Tests/src/Semantic/Conversion_Spec.enso

```diff
@@ -507,6 +507,11 @@ add_specs suite_builder =
 
             foo_vec vec . should_equal [3, 4, 5]
 
+        group_builder.specify "Autoscope Nothing" <|
+            accept (n:Nothing) = n
+
+            accept ..Nothing . is_nothing . should_be_true
```

Comment:
Looks ok but I think I missed why we want to allow that?

Reply:
Addresses #9738